docassemble/ALToolbox/data/static/display_template.css

```diff
@@ -3,3 +3,19 @@
   overflow-y:scroll;
   width:100%;
 }
+/* For the expansion of the copy_button_html() function*/
+textarea {
+  width: 100%;
+  /*The following rule takes effect only with the inline onmouseover event as generated in 
+  display_template() for a specific scenario */  
+  overflow: hidden;
+}
+.al_copy_block {
+  display: block;  
+}
+.al_copy_button span {
+  display: inline-block;
+}
+.al_copy, .card {
```

Comment:
Making this a rule for the `card` class in general seems a little broad. Do we really need it to be for everything with the `card` class? Or is it specific to a display template card?

Reply:
Good point. I'll fix that.